src/retriever.jl

```diff
@@ -1,5 +1,49 @@
-module retriever
+module Retriever
 
-# package code goes here
 
-end # module
+"""
+    datasets()
+
+Return a list of all available dataset scripts.
```

Comment:
Return a list of all available dataset scripts -> Return a list of all available datasets
I think that will sounds better